src/components/dashboard/FeedItems/ListEventItem.js

```diff
@@ -1,41 +1,69 @@
 // @flow
 import React from 'react'
-import { Avatar } from 'react-native-paper'
-import { Text, View } from 'react-native-web'
-import { BigGoodDollar } from '../../common/'
+import { Avatar, withTheme } from 'react-native-paper'
+import { Text } from 'react-native-web'
+import normalize from 'react-native-elements/src/helpers/normalizeText'
+import { BigGoodDollar, Section } from '../../common/'
 import { getFormattedDateTime } from '../../../lib/utils/FormatDate'
 import { listStyles } from './EventStyles'
 import type { FeedEventProps } from './EventProps'
 import EventIcon from './EventIcon'
 import EventCounterParty from './EventCounterParty'
+import getEventSettingsByType from './EventSettingsByType'
+
+const getEventItemStyles = color => ({
+  borderRow: {
+    borderBottomColor: color,
+    borderBottomStyle: 'solid',
+    borderBottomWidth: normalize(2),
+    paddingBottom: normalize(4),
+    marginBottom: normalize(4)
+  },
+  goodDollarAmount: {
+    fontSize: normalize(10),
+    fontFamily: 'Roboto-Bold',
```

Comment:
fontFamily should come from theme. Anyway seems like Fernando PR should take care of this

Reply:
Done

Reply:
seems like you miss thisone